python/cudf/cudf/core/column/column.py

```diff
@@ -2533,6 +2533,14 @@ def _construct_array(
         if inferred_dtype == "interval":
             # Only way to construct an Interval column.
             return pd.array(arbitrary)
+        elif inferred_dtype == "string" and getattr(dtype, "kind", "") == "M":
```

Comment:
Is None a better default?
```suggestion
        elif inferred_dtype == "string" and getattr(dtype, "kind", None) == "M":
```

Reply:
Sure thing. Added